source/apiVolontaria/pages/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from orderable.models import Orderable
+from ckeditor.fields import RichTextField
+
+
+class InfoSection(Orderable):
+
+    is_active = models.BooleanField('Actif', default=True)
+    title = models.CharField('Titre', max_length=255)
+    content = RichTextField('Contenu')
+    is_accordion = models.BooleanField('Accordeon', default=True)
+
+    class Meta(Orderable.Meta):
+        verbose_name = 'Section page informations'
+        verbose_name_plural = 'Sections page informations'
```

Comment:
Please use I18N function: `_('My text')`
On the direct attributes but also on the Meta attributes.

It will help keep trace of all the constant string and allow it to fastly translate the platform.